classes/install.yaml

```diff
@@ -94,10 +88,10 @@ packageScript: |
     {
         installCopy "$@" /usr/ "/usr/include/***" \
             /usr/lib/ \
-            "/usr/lib/*.a" \
-            ${INSTALL_SHARED:+"/usr/lib/*.so*"} \
+            "/usr/lib/***" \
             "/usr/lib/pkgconfig/***" \
             "/usr/lib/cmake/***" \
+            /usr/bin/ "/usr/bin/***" \
```

Comment:
Why include `/bin` for -dev packages?

Reply:
cmake will add the lib and the dll in the <package>-config.cmake file automatically. so if the dll is not there, this will break the build.